django/requirements.txt

```diff
@@ -18,6 +18,7 @@ rules==3.4
 django-cleanup==8.1.0
 django-azure-auth==2.0.0
 gevent==24.2.1
+django-data-fetcher==2.2
```

Comment:
Not used in this PR so I could remove, but will need to use it very shortly for another ticket.